nixos/modules/services/networking/firewall-nftables.nix

```diff
@@ -45,6 +45,18 @@ in
           This option only works with the nftables based firewall.
         '';
       };
+
+      extraReversePathFilterRules = mkOption {
+        type = types.lines;
+        default = "";
+        example = "fib daddr . mark . iif type local accept";
+        description = lib.mdDoc ''
```

Comment:
`lib.mdDoc` is deprecated and the warning lets the manual build fail: https://github.com/NixOS/nixpkgs/pull/305842

Reply:
Right, I knew that, but I must've missed this when copying or such, this was from before the PR removing mdDoc was merged. 
I'll make a follow-up PR to remove it.